examples/react-app/src/FormElements/Sliders.tsx

```diff
@@ -1,86 +1,124 @@
-        <h2>Slider Form</h2>
-        <form name={'sliders'} onSubmit={handleSubmit}>
-          <span>Stepped Slider (5)</span>
-          <PlmgSlider
-            name={'stepped'}
-            marks={true}
-            defaultValue={20}
-            thumbLabel={false}
-            step={5}
-            rangeValues={'0, 20, 50, 70, 100'}
-            onValueUpdated={(e) => handleSlider(e)}
-          />
-          <br />
-          <span>Decimal Slider</span>
-          <PlmgSlider
-            name={'decimal'}
-            thumbLabel
-            rangeValues={'0.1, 0.2, 0.3, 0.4, 0.5'}
-            onValueUpdated={(e) => handleSlider(e)}
-          />
-          <br />
-          <span>Large Value Slider</span>
-          <PlmgSlider
-            name={'large'}
-            step={10000}
-            rangeValues={'100000, 200000, 300000, 400000, 500000'}
-            onValueUpdated={(e) => handleSlider(e)}
-          />
-          <br />
-          <input
-            style={{ height: '20px' }}
-            type={'submit'}
-            value={'Submit Sliders'}
-          />
-        </form>
-      </div>
-      <br />
-      <h2>Single Sliders</h2>
-      <span>Marked Slider</span>
-      <PlmgSlider
-        name={'marked'}
-        marks={true}
-        thumbLabel={false}
-        rangeValues={'0, 10, 20, 30'}
-        onValueUpdated={(e) => handleSlider(e)}
-      />
-      <span>{`Current Marked Slider Value: ${sliders.marked}`}</span>
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        padding: '32px',
+        border: '2px solid #ccc',
+        boxShadow: '0px 0px 10px #ccc',
+        marginTop: '32px',
+        marginBottom: '32px',
+      }}
+    >
+      <form
+        onSubmit={(event) => {
+          event.preventDefault();
+          alert(
+            `You have submitted the form with the following values: ${JSON.stringify(
+              sliders
+            )}`
+          );
+        }}
+      >
+        <h2>Sliders</h2>
+        <label>{SLIDERS.decimal.label}</label>
+        <PlmgSlider
+          name={SLIDERS.decimal.label}
+          default={SLIDERS.decimal.default}
+          marks={SLIDERS.decimal.marks}
+          step={SLIDERS.decimal.step}
+          range={SLIDERS.decimal.range}
+          value={sliders.decimal}
+          onValueUpdated={(event) =>
+            onUpdate(parseFloat(event.detail.value), 'decimal')
+          }
+        />
+        <PlmgButton
+          style={{ paddingTop: '32px' }}
+          label={SLIDERS.decimal.label}
+          design={'outline'}
+          size={'small'}
+          iconLeft={'restartAlt'}
+          color={'neutral'}
+          shadow={false}
+          type={'reset'}
+          onClick={() => {
+            onUpdate(SLIDERS.decimal.default, 'decimal');
+          }}
+        >
+          Reset
+        </PlmgButton>
+        <label>{SLIDERS.stepped.label}</label>
+        <PlmgSlider
```

Comment:
On first render I am seeing thumb label positioning problems.
<img width="1001" alt="Screenshot 2022-09-05 at 03 28 36" src="https://user-images.githubusercontent.com/14997231/188343118-313b38fb-6f98-4380-a9d0-77324622904b.png">
I have seen this happen on all the sliders, it does not appear to be unique to the decimal slider.

Reply:
![Screenshot 2022-09-05 at 09 04 40](https://user-images.githubusercontent.com/104433221/188389381-1cf21a19-56d9-4d24-b1c4-b755bc036939.png)
Managed to recreate the bug by putting a fixed width on the slider (240px).
The input field is actually set by the 'size' value, which is specified by the max value of the input number field. Forcing a min-width is making the overflow the container.
Using an outline is also causing overflow.
Fix: removed min-width, added a negative outline-offset on the input field 
![Screenshot 2022-09-05 at 09 07 20](https://user-images.githubusercontent.com/104433221/188393188-3c58334e-9cbb-4e95-9273-e6cf11b547fd.png)

Reply:
The 'track value' (the input range slider container's) width was getting changed between rendering the component container and calculating the thumb position because of flex gap and fit-content
(changing)[https://github.com/DuckyTeam/Plumage/pull/65/commits/66992c416685617a25b0ad7c94f24346e1a392e2] flex gap for margin-left with a fixed value and removing fit-content prevents input from rendering